acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/timeouts/TimeoutAcceptanceTest.java

```diff
@@ -43,9 +44,10 @@
   @Before
   public void setUp() {
     final NodeConfiguration nodeConfig = new NodeConfigurationBuilder().build();
+    final NodePorts nodePorts = new NodePorts(8545, 8456);
```

Comment:
There is a constant for 8456 as well...

Reply:
NB, discussed in Slack, constant is in Pantheon